sdk/src/Core/Amazon.Runtime/Credentials/Internal/IdentityResolvers/AnonymousIdentityResolver.cs

```diff
@@ -15,14 +15,20 @@
 
 using Smithy.Identity.Abstractions;
 
-namespace Amazon.Runtime.Credentials.Internal
+namespace Amazon.Runtime.Credentials.Internal.IdentityResolvers
 {
-    public class DefaultAWSCredentialsIdentityResolver : IIdentityResolver
+    /// <summary>
+    /// A resolver that provides an anonymous identity. 
+    /// This implementation is used when scheme ID is noAuth, 
```

Comment:
I don't think this line is necessary here (same applies to line 30 of `DefaultAWSCredentialsIdentityResolver`); the resolvers don't care about auth schemes (and I feel like if they ever change we will forget to update this comment).